cmd/kms-handlers.go

```diff
@@ -168,7 +168,20 @@ func (a kmsAPIHandlers) KMSVersionHandler(w http.ResponseWriter, r *http.Request
 
 // KMSCreateKeyHandler - POST /minio/kms/v1/key/create?key-id=<master-key-id>
 func (a kmsAPIHandlers) KMSCreateKeyHandler(w http.ResponseWriter, r *http.Request) {
+	// If env variable MINIO_KMS_SECRET_KEY is populated, prevent creation of new keys
 	ctx := newContext(r, w, "KMSCreateKey")
+	if GlobalKMS != nil {
+		status, err := GlobalKMS.Stat(ctx)
```

Comment:
```suggestion
		status, err := GlobalKMS.Stat(ctx)
```

why do we need to do an extra network call here? isn't it easier to use something like `

```
GlobalKMS.IsLocal()
```

It is very easier

```
if GlobalKMS.IsLocal() {
       writeErrorResponseJSON(ctx, w, errorCodes.ToAPIErr(ErrKMSDefaultKeyAlreadyConfigured), r.URL)
       return
}